samples/unit-tests/series/marker/demo.js

```diff
@@ -181,6 +181,31 @@ QUnit.test('Marker size and position', function (assert) {
         series.stateMarkerGraphic.attr('class'),
         '#5430: State marker should have class set'
     );
+
+    series.update({
+        marker: {
+            enabled: true,
+            width: 25,
+            height: 25
+        }
+    });
+
+    series.update({
+        marker: {
+            width: 50,
+            height: 50
+        }
+    });
```

Comment:
In order to test this functionality we need to: 

- Have markers enabled, with a given width/height
- Update ONLY width and height
- Check if update got triggered

Changing the `enabled` property updated the series itself, which also changed the width/height. That's why there are two updates. We couldn't do it in one go if markers were previously disabled in previous tests.